README.md

```diff
@@ -134,6 +134,7 @@ This provider exposes quite a few provider-specific configuration options:
 * `subnet_id` - The subnet to boot the instance into, for VPC.
 * `associate_public_ip` - If true, will associate a public IP address to an instance in a VPC.
 * `tags` - A hash of tags to set on the machine.
+* `package_tags` - A hash of tags to set on the ami generated during the package operation.
```

Comment:
Well, it IMO depends how it's **actually** implemented, if there would be something similar to AMI, I'd call it probably `image_tags` because AMI btw. stands for Amazon Machine **Image**, not package. But as long as this is `vagrant-aws` plugin, there will be most likely no other similar *images* and more importantly that's the way it's [implemented](https://github.com/mitchellh/vagrant-aws/pull/334/files#diff-00ba837fd1472b2fa143e0cb416ec1e7R62) at the end.

I feel like it's an extra abstraction layer which will only exist in namings, but not in the code.
Maybe I'm just overlooking something?